core/src/lib.rs

```diff
@@ -1,5 +1,11 @@
+mod block_id;
 mod block_number;
 mod number_visitor;
+mod timestamp;
```

Comment:
@gostkin it's not visible on the github UI, but the file is actually named `Timestamp.rs`. This means this doesn't compile in case-sensitive filesystems (like linux).

Reply:
I believe it was renamed in the last commit, could you check pls?

Reply:
https://github.com/dcSpark/dcspark-core/pull/5/commits/ff166d3922f423102f18b300ae51b0c38ae18a4c#diff-1c0b2d62287247d90db6a75d7df18fa639652e07cd96ac29bb377a753d0d0906